doc/admin/predeployment_check.md

```diff
@@ -29,16 +29,79 @@ $ sudo reboot
 ```
 
 !!! warning
-    VFIO support is a new feature for DAOS 1.2 and has been tested on the
-    following platforms:
-    •	CentOS 7.7
+    VFIO support is a new feature for DAOS 1.2 and has been tested on CentOS 7.7
 
 ## Time Synchronization
 
 The DAOS transaction model relies on timestamps and requires time to be
 synchronized across all the storage and client nodes. This can be done
 using NTP or any other equivalent protocol.
 
+## User/Group Synchronization
+
+DAOS does not require user and group to be synchronized between client and
+and storage nodes.
+DAOS ACL stores the actual user and group name (instead of numeric ID) and does
+not verify group membership check on the storage node. The DAOS agent (running
+on the client nodes) is responsible for converting the UID/GID to name and to
+resolve the list of groups that the users belong to. This informations are then
+signed by the agent and send to the storage node.
```

Comment:
Suggested update: "DAOS ACLs store the actual user and group names (instead of numeric IDs), and therefore the servers do not need access to a synchronized user/group database. The DAOS Agent (running on the client nodes) is responsible for resolving UID/GID to user/group names which are added to a signed credential and sent to the DAOS storage nodes."